### DISCUSSION TITLE
Nextjs FullStack project

### DISCUSSION THREAD

**Discussion by Racine14Creator** [community/community#119786] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

# The best window... 
## I need someone to work with one this project... For free! I said for free.

![windows](https://github.com/community/community/assets/148460524/8dbf112c-7791-4ef4-9943-c34ec99d2a5a)

**Comment by LiteBrite82** [comment, marked as answer]
Hi @Racine14Creator , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
